arm/Microsoft.Automation/automationAccounts/readme.md

```diff
@@ -49,7 +49,43 @@ This module deploys an Azure Automation Account.
 | `tags` | object | `{object}` |  | Optional. Tags of the Automation Account resource. |
 | `userAssignedIdentities` | object | `{object}` |  | Optional. The ID(s) to assign to the resource. |
 | `variables` | _[variables](variables/readme.md)_ array | `[]` |  | Optional. List of variables to be created in the automation account. |
-| `workspaceId` | string |  |  | Optional. Resource ID of log analytics. |
+| `workspaceId` | string |  |  | Optional. Resource identifier of log analytics. |
```

Comment:
Should not be rolled back. Please update in bicep file and here

Reply:
done